gnome-initial-setup/pages/password/gis-password-page.c

```diff
@@ -320,6 +347,14 @@ gis_password_page_constructed (GObject *object)
 
   validate (page);
 
+  priv->input_source_manager = g_dbus_proxy_new_for_bus_sync (G_BUS_TYPE_SESSION,
```

Comment:
It occurred to me that we don't actually need to keep this object around, and we could instead use `g_dbus_connection_call_sync()` when setting the property, but this is fine too.